docs/release-notes/8.2.asciidoc

```diff
@@ -1,4 +1,34 @@
 [[ecs-release-notes-8.2.0]]
 === 8.2.0
 
-coming[8.2.0]
+[[schema-changes-8.2.0]]
+[float]
+==== Schema changes
+
+[[schema-added-8.2.0]]
+[float]
+===== Added
+
+* Add beta `container.*` metric fields. {ecs_pull}1789[#1789]
+* Add six new syslog fields to `log.syslog.*`. {ecs_pull}1793[#1793]
+* Added `faas.id`, `faas.name` and `faas.version` fields as beta. {ecs_pull}1796[#1796]
+* Added linux event model beta fields and reuses to support RFC 0030. {ecs_pull}1842[#1842], {ecs_pull}1847[#1847], {ecs_pull}1884[#1884]
+* Added `threat.feed.dashboard_id`, `threat.feed.description`, `threat.feed.name`, `threat.feed.reference` fields. {ecs_pull}1844[#1844]
+
+#### Improvements
```

Comment:
Wasn't sure if this PR was ready for review, but I noticed the headings aren't rendering quite right yet.

<img width="757" alt="Screen Shot 2022-05-02 at 2 24 03 PM" src="https://user-images.githubusercontent.com/7226265/166312912-d778ecf8-024d-453a-875a-df96d9b2493b.png">

Reply:
Good catch, looks like I missed committing some changes

Reply:
@ebeahan all better now?